test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -261,7 +261,7 @@ string TestFunctionCall::formatBytesRange(
 		os << BytesUtils().formatHexString(_bytes);
 		break;
 	case ABIType::String:
-		os << BytesUtils().formatString(_bytes);
+		os << BytesUtils().formatString(_bytes, _abiType.size);
```

Comment:
Why do we cut it off?

Reply:
We apply the cut-off such that [61, 62, 63, 00, .., 00] formats nicely to "abc".